src/Analyzers/CSharp/Tests/UseDeconstruction/UseDeconstructionTests.cs

```diff
@@ -3,818 +3,914 @@
+                }
+                """;
+
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact]
         public async Task TestNotIfTupleMethodIsUsed()
         {
-            await TestMissingInRegularAndScriptAsync(
-@"class C
-{
-    void M()
-    {
-        var [|t1|] = GetPerson();
-        Console.WriteLine(t1.ToString());
-    }
+            var code = """
+                class C
+                {
+                    void M()
+                    {
+                        var t1 = GetPerson();
+                        System.Console.WriteLine(t1.ToString());
+                    }
+
+                    (string name, int age) GetPerson() => default;
+                }
+                """;
 
-    (string name, int age) GetPerson() => default;
-}");
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact]
         public async Task TestNotIfTupleDefaultElementNameUsed()
         {
-            await TestMissingInRegularAndScriptAsync(
-@"class C
-{
-    void M()
-    {
-        var [|t1|] = GetPerson();
-        Console.WriteLine(t1.Item1);
-    }
+            var code = """
+                class C
+                {
+                    void M()
+                    {
+                        var t1 = GetPerson();
+                        System.Console.WriteLine(t1.Item1);
+                    }
+
+                    (string name, int age) GetPerson() => default;
+                }
+                """;
 
-    (string name, int age) GetPerson() => default;
-}");
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact]
         public async Task TestNotIfTupleRandomNameUsed()
         {
-            await TestMissingInRegularAndScriptAsync(
-@"class C
-{
-    void M()
-    {
-        var [|t1|] = GetPerson();
-        Console.WriteLine(t1.Unknown);
-    }
+            var code = """
+                class C
+                {
+                    void M()
+                    {
+                        var t1 = GetPerson();
+                        System.Console.WriteLine(t1.{|CS1061:Unknown|});
+                    }
 
-    (string name, int age) GetPerson() => default;
-}");
+                    (string name, int age) GetPerson() => default;
+                }
+                """;
+
+            await VerifyCS.VerifyCodeFixAsync(code, code);
         }
 
         [Fact]
         public async Task TestTrivia1()
         {
-            await TestInRegularAndScript1Async(
-@"class C
-{
-    void M()
-    {
-        /*1*/(/*2*/int/*3*/ name, /*4*/int/*5*/ age)/*6*/ [|t1|] = GetPerson();
-        Console.WriteLine(/*7*/t1.name/*8*/ + "" "" + /*9*/t1.age/*10*/);
-    }
+            await VerifyCS.VerifyCodeFixAsync("""
+                class C
+                {
+                    void M()
+                    {
+                        /*1*/(/*2*/string/*3*/ name, /*4*/int/*5*/ age)/*6*/ [|t1|] = GetPerson();
```

Comment:
This test case is for trivia, so I consider tuple type mismatch here an unintended noise error